### ISSUE TITLE
got message "Lefthook wasn't build for x86_64-linux"

### ISSUE THREAD

**Issue by cbliard** [evilmartians/lefthook#278]
After upgrading to version 1.0.0, `lefthook install` gives me the following error:

```
/home/cbliard/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/lefthook-1.0.0/bin/lefthook:34:in `<top (required)>': Invalid platform. Lefthook wasn't build for x86_64-linux (RuntimeError)
	from /home/cbliard/.asdf/installs/ruby/3.1.2/bin/lefthook:25:in `load'
	from /home/cbliard/.asdf/installs/ruby/3.1.2/bin/lefthook:25:in `<main>'
```

The path being looked for is `/home/cbliard/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/lefthook-1.0.0/libexec/lefthook_linux_amd64_v1/lefthook`, but it does not exist. `/home/cbliard/.asdf/installs/ruby/3.1.2/lib/ruby/gems/3.1.0/gems/lefthook-1.0.0/libexec/lefthook_linux_amd64/lefthook` without the `_v1` suffix actually exists.

The `_v1` suffix was introduced in fa2403ec942.

Replacing the line
```
suffix = arch == "amd64" ? "_v1" : "" # GOAMD64
```
by
```
suffix = ""
```
fixes it for my case.

**Comment by muuvmuuv** [comment]
@mrexox the GitHub release is missing to be able to download the binary via the installer ^^

**Comment by mrexox** [comment]
Thank you, @muuvmuuv ! I have created a GitHub release, so it should be working now for the installer too.

**Comment by cbliard** [comment]
Indeed, the issue is fixed now with version 1.0.1. Thanks a lot for the quick fix :heart:
